src/components/Apps/AppsList/AppsList.tsx

```diff
@@ -46,13 +73,15 @@ const AppsList: React.FC = () => {
   const allApps = useSelector(selectApps);
   const selectedCatalogs = useSelector(selectSelectedCatalogs);
   const memoSelectedCatalogs = selectedCatalogs.join('');
-  const sortOrder = useSelector(selectAppSortOrder);
 
   const apps = useMemo(() => searchApps(debouncedSearchQuery, allApps), [
     debouncedSearchQuery,
     memoSelectedCatalogs,
   ]);
 
+  const sortOrder = useSelector(selectAppSortOrder);
+  apps.sort(sortFuncs[sortOrder]);
+
```

Comment:
But why not do the sorting inside of the `useMemo` function argument?

I believe this should do it:

```suggestion
  const sortOrder = useSelector(selectAppSortOrder);

  const apps = useMemo(() => {
    const appCollection = searchApps(debouncedSearchQuery, allApps);
    appCollection.sort(sortFuncs[sortOrder]);
    
    return appCollection;
  }, [
    debouncedSearchQuery,
    memoSelectedCatalogs,
    sortOrder,
  ]);

```